src/features/Navigation/components/Navigation.elements.tsx

```diff
@@ -1,36 +1,65 @@
-import { styled } from "@mui/material/styles";
+import {
+    CSSObject,
+    styled
+} from "@mui/material/styles";
 import {
     AppBar,
     Container,
     ContainerProps,
     Drawer,
-    List
+    List,
+    ListItemIconProps
 } from "@mui/material";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import * as React from "react";
 
 export const TOP_MARGIN = 20;
-const drawerWidthOpen = 240;
-const drawerWidthClosed = 58;
+const drawerWidth = 240;
 
 export const Header = styled(AppBar)(({theme}) => ({
     zIndex: theme.zIndex.drawer + 10,
     backgroundColor: theme.palette.primary.main,
     height: 5,
 }));
 
-export const Sidebar = styled(Drawer, { shouldForwardProp: (prop) => prop !== "open" })(
-        ({theme, open}) => ({
-            // open ? openedMixin(theme) : closedMixin(theme),
-            "& .MuiDrawer-paper": {
-        width: open ? drawerWidthOpen : drawerWidthClosed,
-        borderRightWidth: 0,
-                overflowX: "hidden",
-                height: "100%",
-                transition: theme.transitions.create("width", {
-            easing: theme.transitions.easing.sharp,
-            duration: open? theme.transitions.duration.leavingScreen : theme.transitions.duration.enteringScreen,
+const openedMixin = (theme): CSSObject => ({
```

Comment:
What is the reason behind adding these mixins? I know the examples have them on the MUI documentation but they felt silly to me. I am curious why you feel like it's better.

Reply:
It avoided doing "closed size" math, and allowed the content to reflow correctly, since they're applied to both the drawer and its Paper. They were effectively there already, inlined into `MainDesktop`, but didn't do everything from that location.